qutebrowser/browser/network/networkmanager.py

```diff
@@ -241,12 +242,22 @@ def clear_rejected_ssl_errors(self, url):
     @pyqtSlot('QNetworkReply', 'QAuthenticator')
     def on_authentication_required(self, reply, authenticator):
         """Called when a website needs authentication."""
-        answer = self._ask("Username ({}):".format(authenticator.realm()),
-                           mode=usertypes.PromptMode.user_pwd,
-                           owner=reply)
-        if answer is not None:
-            authenticator.setUser(answer.user)
-            authenticator.setPassword(answer.password)
+        user, password = None, None
+        if not hasattr(reply, "_netrc_used"):
```

Comment:
Please call the attribute `netrc_used`, i.e. without the `_` prefix - `_foo` attributes are usually considered "private", i.e. not intended to be accessed from outside that class.